<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPodt</class>
 <widget class="QWidget" name="FormPodt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Подтяжки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>С</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DbDateEdit" name="dateEditBeg"/>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>По</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DbDateEdit" name="dateEditEnd"/>
         </item>
         <item>
          <widget class="QPushButton" name="cmdUpd">
           <property name="toolTip">
            <string>Обновить список подтяжек</string>
           </property>
           <property name="text">
            <string>Обновить</string>
           </property>
           <property name="icon">
            <iconset theme="view-refresh"/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="DbViewer" name="tableView">
         <property name="minimumSize">
          <size>
           <width>540</width>
           <height>0</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Только марка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DbComboBox" name="comboBoxFMark">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Ф</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DbComboBox" name="comboBoxFDiam"/>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Тип</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DbComboBox" name="comboBoxFType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="4">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>След. волочение</string>
           </property>
          </widget>
         </item>
         <item row="0" column="8" colspan="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>68</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Исх. парт.</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Оборудование</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="DbComboBox" name="comboBoxLine">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Диам. гот.</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="DbComboBox" name="comboBoxDiam"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Комментарий</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="9">
          <widget class="QLineEdit" name="lineEditComm"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Номер</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditNum">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Дата</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="DbDateEdit" name="dateEdit"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Тип</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5" colspan="3">
          <widget class="DbComboBox" name="comboBoxType">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="9">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>68</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="5" colspan="4">
          <widget class="DbComboBox" name="comboBoxVol"/>
         </item>
         <item row="1" column="1" colspan="9">
          <widget class="DbComboBox" name="comboBoxSrs"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBoxCont">
           <property name="title">
            <string>Поступление</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="DbViewer" name="tableViewCont"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxPodtIn">
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Приход</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="DbViewer" name="tableViewPodtIn"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBoxVol">
           <property name="title">
            <string>Волочение</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="DbViewer" name="tableViewVol"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxPodtOut">
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Расход</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="DbViewer" name="tableViewPodtOut"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Партии готовой проволоки</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="TableView" name="tableViewPart">
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxDef">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Брак</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="DbViewer" name="tableViewPodtDef"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutBtn">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>olap/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>DbViewer</class>
   <extends>QTableView</extends>
   <header>db/dbviewer.h</header>
  </customwidget>
  <customwidget>
   <class>DbComboBox</class>
   <extends>QComboBox</extends>
   <header>db/dbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>DbDateEdit</class>
   <extends>QDateEdit</extends>
   <header>db/dbdateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditNum</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>comboBoxSrs</tabstop>
  <tabstop>comboBoxLine</tabstop>
  <tabstop>comboBoxDiam</tabstop>
  <tabstop>tableViewCont</tabstop>
  <tabstop>tableViewPart</tabstop>
  <tabstop>dateEditBeg</tabstop>
  <tabstop>dateEditEnd</tabstop>
  <tabstop>cmdUpd</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
